package.json

```diff
@@ -30,6 +30,7 @@
     "next-auth": "^4.12.3",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-router-dom": "^6.18.0",
```

Comment:
I think we can uninstall `react-router-dom` and opt-in for `useRouter` that is provided by Nextjs.13